esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -482,6 +483,7 @@ define(
         NotActive: "Inacivado",
         NothingSelected: "Nada Seleccionado",
         NotInSuperfiles: "No en las Superfiles",
+        NotSuspended: "No esta suspendido",
```

Comment:
depends on subject, if the subject is considered female this would have to be "No esta suspendidA" 
queries (consultas) would be female

Reply:
i think here it would be in reference to queries as mentioned so i think "no esta suspendida" would be correct

Reply:
@miguelvazq can you clarify?

Reply:
`Is not suspended` is the context here.

Reply:
@miguelvazq can you confirm we're referencing queries? if so, all these "suspended" messages might need to change to some form of Suspendida (since queries are consultAs)